testing/loki_alerts/fluent-bit.trailer.conf

```diff
@@ -0,0 +1,44 @@
+# Deal specifically with some log parsing initially
+@include /fluent-bit/etc/couchbase/filter-handle-logfmt.conf
+
+# Add in common info
+@include /fluent-bit/etc/couchbase/filter-add-common-info.conf
+
+# Deal with missing/incorrect level & filename information
+@include /fluent-bit/etc/couchbase/filter-handle-levels.conf
+@include /fluent-bit/etc/couchbase/filter-handle-filenames.conf
+
+# Optionally include this to get common problems duplicated to a specific stream
+# @include /fluent-bit/etc/couchbase/filter-common-problems.conf
+
+# TEST SPECIFIC: Loki only lets us query a 12 hour window at most.
+# Rather than parsing all the logs _again_ to determine when they start,
+# we'll remap the timestamps in the logs to start from now-ish.
+[FILTER]
+    Name lua
+    Match *
+    Script rebase_times.lua
```

Comment:
Pretty cool, be good to get that in the official docs as an example of LUA + replay